_projects/2024/100535241/100535241.Rmd

```diff
@@ -0,0 +1,824 @@
+---
+title: "Winners and losers of tax legislation changes"
+description: |
+  Recreating the ABC news article Graph for MCSS Data Viz 24/25
+categories: "2024"
+author: Brad McKenzie
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+    toc_float: true
+    code_folding: true
+---
+
+# About the graph
+
+I have recreated the following graph from [this February 2024 article by ABC news Australia](https://www.abc.net.au/news/2024-02-02/biggest-winners-and-losers-of-revised-stage-3-tax-cuts/103415830).
+
+```{r original-plot, echo=FALSE, fig.cap="Original graph from ABC article", layout="l-body-outset"}
+knitr::include_graphics("~/Documents/UC3M - coursework/Data Vizualisation/dataviz/ABC_tax_income_plot.png")
```

Comment:
This graph should be included in the PR close to this Rmd, and then this should be just `knitr::include_graphics("ABC_tax_income_plot.png")`